common/build.gradle

```diff
@@ -153,6 +155,11 @@ dependencies {
     testImplementation "org.powermock:powermock-classloading-xstream:$rootProject.ext.powerMockVersion"
     testImplementation "org.robolectric:robolectric:$rootProject.ext.robolectricVersion"
     testImplementation "androidx.test:core:$rootProject.ext.androidxTestCoreVersion"
+    // Junit5 - we can enable it if we discover that desugaring allows it to run on API<26
```

Comment:
I'll leave this up to you: I usually try to steer away from committing commented-out bits - your call if leaving this in/out is best

Reply:
I'm OK not doing it. I just wish there was a place to put it where we can remember it.